website/i18n/en.json

```diff
@@ -161,6 +161,15 @@
       "interactionmanager": {
         "title": "InteractionManager"
       },
+      "intro-react-native-components": {
+        "title": "Core Components and Native Components"
+      },
+      "intro-react": {
+        "title": "Key React Concepts"
```

Comment:
tbh I feel that even without "Key" in front, this title would work.

Reply:
At first it was "React Basics" to imply that it wasn't covering ALL of React. But then we wondered if there was a more inclusive way of expressing "just enough React to get by but not all the React." What do you think?

Reply:
Fundamentals of React?
React foundations?

Reply:
React Fundamentals